lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
@yanenkoa I see. In fact, it is pretty heavy-weight so we probably don't want it to be loaded by default. In the real life we'll load it when user hits a key (or whatever). So, I think it need to be loaded similar to extensions/themes/modes/snippets and other such stuff. See Makefile.dryice.js, lines from 340 and below. Probably extension is fine for katex.